packages/mobile/src/harmony-native/components/TextLink/TextLink.tsx

```diff
@@ -107,3 +139,59 @@ export const TextLink = <ParamList extends ReactNavigation.RootParamList>(
     return <TouchableOpacity {...rootProps}>{element}</TouchableOpacity>
   }
 }
+
+/**
```

Comment:
maybe move text-link-animation and text-link-flowing next to eachother?

Reply:
now that im thinking about the code more, my guess is that we can drop end-adornment from text-link, and just have user-link handle the end-adornment + opacity (or nothing if that's what design wants) for the link part and the end-adornment part?